src/ArmTemplates/Commands/Executors/ExtractorExecutor.cs

```diff
@@ -1281,5 +1281,39 @@ async Task<List<string>> GetApiNamesToExtract(string singleApiName, List<string>
 
             return apiDictionary;
         }
+
+        async Task<Dictionary<string, List<string>>> GetAllAPIsDictionaryByVersionSetName(ExtractorParameters extractorParameters)
```

Comment:
maybe we can move that to apisClient? To have a possibility to use this method in other places as well.
I am thinking of removing any unneccesary methods from `ExtractorExecutor`

Reply:
I don't think apisClient is the best place for this kind of methods, I will create additional class apisClientUtilies to hold this kind of data manipulations there :) what do you think?

Reply:
ye, we can create another place.
I am just curious if we need this API in `ExtractorExecutor`

Reply:
Not exactly, I will check this one will update PR or answer in comment thread. 
Thank you for the review!

Reply:
Moved to different method class. Thank you for the review